common/locales/en/settings.json

```diff
@@ -108,10 +108,12 @@
     "emailNotifications": "Email Notifications",
     "wonChallenge": "You won a Challenge!",
     "newPM": "Received Private Message",
+    "newPMInfo": "New Message from <%= name %>: <%= message %>",
     "sentGems": "Sent gems!",
     "giftedGems": "Gifted Gems",
-    "giftedGemsInfo": "<%= amount %> Gems - by <%= name %>",
+    "giftedGemsInfo": "<%= name %> gifted you <%= amount %> Gems",
     "giftedSubscription": "Gifted Subscription",
+    "giftedSubscriptionInfo": "<%= name %> gifted you a <%= months %> Subscription",
```

Comment:
Ah nevermind, I see where the strings are used.  But maybe the string changes/additions should still be a separate (mini) PR for clarity.